transport/http/config.go

```diff
@@ -81,21 +81,50 @@ func (ts *transportSpec) Spec() config.TransportSpec {
 // 	transports:
 // 	  http:
 // 	    keepAlive: 30s
+//      maxIdleConnsPerHost: 2
+//      connTimeout: 500ms
+//      connBackoff:
+//        exponential:
+//          min: 100ms
+//          base: 100ms
+//          max: 30s
```

Comment:
Changed to (first: 10ms, max: 30s). Connection timeout is still 500ms, but we can revisit.